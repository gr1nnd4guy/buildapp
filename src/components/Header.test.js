import React from "react";
import { render } from '@testing-library/react';
import Header from "./header";

test('renders header with title', () => { 
    const { container } = render(<Header title="My App"/>);
    expect(container).toMatchSnapshot();
  });