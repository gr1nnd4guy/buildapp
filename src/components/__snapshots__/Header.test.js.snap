// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header with title 1`] = `
<div>
  <header>
    <h1>
      My App
    </h1>
  </header>
</div>
`;
